src/main/resources/com/mathworks/ci/RunMatlabTestsStep/help-useParallel.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    <p>Whether to run tests in parallel. This feature requires a Parallel Computing Toolbox license, and might not be 
+      compatible with other options. If incompatible, testing occurs in serial regardless of the value of this feature.</p> 
```

Comment:
Consider replacing "regardless of the value of this feature." with "regardless of the specified value."